library/core/src/main/java/com/google/android/exoplayer2/ExoPlayer.java

```diff
@@ -684,4 +684,41 @@ public ExoPlayer build() {
    * @see #experimentalSetOffloadSchedulingEnabled(boolean)
    */
   boolean experimentalIsSleepingForOffload();
+
+  /**
+   * Toggle live catchup playback.
+   * @param state True to enable live catchup playback.
+   */
+  default void setEnableLivePlaybackSpeedControl(boolean state) {}
```

Comment:
Can we just leve the methods in LivePlaybackSpeedControl?

Reply:
Yes I can replace them with `getLiveConfiguration().setLivePlaybackSpeedControl()`